pages/common/jj.md

```diff
@@ -0,0 +1,25 @@
+# jj
+
+> Jujutsu, a version control system.
+> Some subcommands such as `log`, `desc`, `new`, `git`, etc. have their own usage documentation.
+> More information: <https://jj-vcs.github.io/jj/latest/cli-reference/>.
+
+- Update description of the revisions specified by given revsets (e.g. `B::D`, `A..D`, `B|C|D`, etc.):
+
+`jj describe {{[-r|--revision]}} {{revsets}}`
+
+- Create a new commit/revision on top of the revisions specified by the given revsets:
+
+`jj new {{revsets}}`
```

Comment:
Since users will want to specify exactly one commit most of the time, should this be singular `jj new {{revset}}`? Are readers expected to know that you can create merge commits by passing multiple revisions?

Reply:
+1, this should definitely be called out